cpu/stm32f7/Makefile.include

```diff
@@ -1,5 +1,28 @@
 export CPU_ARCH = cortex-m7
 export CPU_FAM  = stm32f7
 
+# STM32F7 uses sectors instead of pages, where the minimum sector length is 16KB or
+# 32kB (the first sector), depending on the CPU_MODEL. Therefore RIOTBOOT_LEN must
+# be 16KB or 32kB to cover a whole sector.
+ifneq (,$(filter stm32f722ze ,$(CPU_MODEL)))
```

Comment:
This is a specific case for stm32f722ze so I would either add riotboot for nucleo-f722ze or remove this part. I prefer the former so I'll test on the board and report the result in this PR.

Reply:
It is not really a specific for `stm32f722ze`, a lot of `stm32f7` have 16kB sector. It's just that we only support 4 `f7`, and the other 3 have 32kB sized sectors.

All `stm32f7x2xx` hace 16kB and all `stm32f76xxx` that are smaller than 2Mb. It really depends on the `stm32f7` sub-family.

Reply:
Sorry, but if you add this if, you add a case for stm32f722ze.

Anyway, I tested this PR on nucleo-f722ze and it works. So you can add the riotboot feature for this board as well.

Reply:
Ok, should I leave the if as it is? The other option would be to add defines for the `stm32f7` sub-families.

Reply:
>  should I leave the if as it is?

Yes, but add then just add riotboot feature to nucleo-f722ze and I'll ACK this PR.

Reply:
In that case do you have stm32769i-disco so we add it for all stm32f7 board in one go?

Reply:
Unfortunately, we don't have this one.

Reply:
Ok, I didn't include it then.